daemon/containerd/image_builder.go

```diff
@@ -200,7 +200,7 @@ func newROLayerForImage(ctx context.Context, imgDesc *ocispec.Descriptor, i *Ima
 		key:                key,
 		c:                  i.client,
 		snapshotter:        i.snapshotter,
-		diffID:             digest.Digest(parent),
+		diffID:             "",
 		contentStoreDigest: "",
```

Comment:
I explicitly kept it, because that's the desired value due to the nature of the read-only layer. It also stops you from wondering in the future if this was omitted on purpose or by accident 😄

Reply:
Gotcha! Should we leave a `// comment` after both of them to document that? ("must be empty, because ...") ? Without a comment, I can see myself (or others) removing those lines some day in the future.